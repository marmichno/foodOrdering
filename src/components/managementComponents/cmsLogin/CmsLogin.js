import {Link} from 'react-router-dom';

export const CmsLogin = () =>{
    return(
        <div className="managementLoginMainContainer">
            <div className="loginBox">
                <label for="category">
                    <span className="contentName"><p>Login</p></span>
                </label>
                <input type="text" autocomplete="off" name="category" className="firstSentenceInput" required></input>

                <label for="category">
                    <span className="contentName"><p>Password</p></span>
                </label>
                <input type="password" autocomplete="off" name="category" className="firstSentenceInput" required></input>

                <Link to={{
                        pathname:'/cmsLogin/cms'
                    }}>
                <button>Login</button>
                </Link>
            </div>
        </div>
    )
}