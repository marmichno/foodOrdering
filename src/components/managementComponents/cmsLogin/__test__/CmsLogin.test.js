import {CmsLogin} from '../CmsLogin';
import {BrowserRouter} from 'react-router-dom';
import TestRenderer from "react-test-renderer";
import "@testing-library/jest-dom/extend-expect";

test("matches snapshot", () => {
    const tree = TestRenderer.create(<BrowserRouter><CmsLogin/></BrowserRouter>).toJSON();
    expect(tree).toMatchSnapshot();
});