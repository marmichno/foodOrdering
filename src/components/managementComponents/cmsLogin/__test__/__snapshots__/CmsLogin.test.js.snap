// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<div
  className="managementLoginMainContainer"
>
  <div
    className="loginBox"
  >
    <label
      for="category"
    >
      <span
        className="contentName"
      >
        <p>
          Login
        </p>
      </span>
    </label>
    <input
      autocomplete="off"
      className="firstSentenceInput"
      name="category"
      required={true}
      type="text"
    />
    <label
      for="category"
    >
      <span
        className="contentName"
      >
        <p>
          Password
        </p>
      </span>
    </label>
    <input
      autocomplete="off"
      className="firstSentenceInput"
      name="category"
      required={true}
      type="password"
    />
    <a
      href="/cmsLogin/cms"
      onClick={[Function]}
    >
      <button>
        Login
      </button>
    </a>
  </div>
</div>
`;
